src/webview/html/messageAsHtml.js

```diff
@@ -85,15 +85,28 @@ export const flagsStateToStringList = (flags: FlagsState, id: number): string[]
 export default (backgroundData: BackgroundData, message: Message | Outbox, isBrief: boolean) => {
   const { id, timestamp } = message;
   const flagStrings = flagsStateToStringList(backgroundData.flags, id);
+  const isUserMuted = !!message.sender_id && backgroundData.mutedUsers.has(message.sender_id);
+
   const divOpenHtml = template`
     <div
-     class="message ${isBrief ? 'message-brief' : 'message-full'}"
+     class="message ${isBrief ? 'message-brief' : 'message-full'} ${
+    isUserMuted ? 'message-muted' : ''
+  }"
      id="msg-${id}"
      data-msg-id="${id}"
      $!${flagStrings.map(flag => template`data-${flag}="true" `).join('')}
     >`;
   const messageTime = shortTime(new Date(timestamp * 1000), backgroundData.twentyFourHourTime);
 
+  // TODO: i18n
```

Comment:
Is your intention to fix this TODO before merge? If so, always good to mention that in the commit's summary line and/or the PR thread.

Reply:
It's my intention to fix this before we launch this feature, but I think we could merge an initial version of this before that.

Currently this is the same as the status quo as for widgets, which maybe isn't a good excuse, but does make more more comfortable merging it before this is done. I'm happy to block on this if you'd prefer, though.

Reply:
Added i18n in one of the more recent commits :)